### PR TITLE
fix increment-decrement lint error

### PR DESCRIPTION
**Pull request by danielqsj**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

fix increment-decrement lint error:
1. replace `+=1` with `++`
2. replace `-=1` with `--`

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apimachinery/pkg/api/resource/math.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func negativeScaleInt64(base int64, scale Scale) (result int64, exact bool) {
 	}
 	if fraction {
 		if base > 0 {
-			value += 1
+			value++
 		} else {
 			value += -1
```
[the remark this answers is not in the document]
Comment (oomichi) [reply staging/src/k8s.io/apimachinery/pkg/api/resource/math.go]: +1

[the remark this answers is not in the document]
Comment (danielqsj) [reply staging/src/k8s.io/apimachinery/pkg/api/resource/math.go]: Great catch. Fixed, thanks.

**Comment by danielqsj** [comment]
friendly ping @liggitt  for approve :)

**Pull request merged**